src/main/java/org/opentripplanner/middleware/utils/ItineraryUtils.java

```diff
@@ -93,4 +99,139 @@ public static String toQueryString(Map<String, String> params) {
         result.put(IGNORE_REALTIME_UPDATES_PARAM, "true");
         return result;
     }
+
+    /**
+     * TODO: fully implement
+     */
+    public static boolean itineraryIsSavable(Itinerary itinerary) {
+        return true;
+    }
+
+    /**
+     * Returns true if the itineraries match for the purposes of trip monitoring.
+     *
+     * @param previousItinerary The original itinerary that others are compared against.
+     * @param newItinerary A new itinerary that might match the previous itinerary.
+     */
+    public static boolean itinerariesMatch(Itinerary previousItinerary, Itinerary newItinerary) {
```

Comment:
As far as param names, I'm somewhat partial to the language used above `referenceItinerary` and `itineraryCandidate`. I'd be ok with other names too, but I think the previous/new combo doesn't quite match. Even the name here in the javadoc (original itinerary) might be a little better.

Reply:
Fixed in ffc3b49b3ad576d936b04758d572efdf3f217ab2